.github/workflows/pyopenms-wheels.yml

```diff
@@ -191,8 +193,8 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE="Release" -DOPENMS_CONTRIB_LIBS="$GITHUB_WORKSPACE/contribbld" -DCMAKE_PREFIX_PATH="$(echo $Qt5_Dir)/lib/cmake;${Qt5_Dir}" -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 -DBOOST_USE_STATIC=ON ../OpenMS
           make -j4 OpenMS SuperHirn
 
-          # Search for the latest 3 main python versions in the repository pkgs/main
-          SUPPORTED_PYTHONS=$(conda search --full-name python | grep "pkgs/main" | awk '{print $2}' | cut -d '.' -f -2 | uniq | tail -3)
+          # Search for the latest 4 main python versions in the repository pkgs/main
+          SUPPORTED_PYTHONS=$(conda search --full-name python | grep "pkgs/main" | awk '{print $2}' | cut -d '.' -f -2 | uniq | tail -4)
```

Comment:
this is not needed: Here it basically loops over the python versions.

Reply:
ok so I can remove all these lines here and below?